### DISCUSSION TITLE
[1451940]主播女孩重度依赖

### DISCUSSION THREAD

**Discussion by L1keava** [pjy612/SteamManifestCache#796] [Share]
### Steam 游戏ID

1451940

### 密钥列表

```hash
1451942;058fdb113770acdb9ebefce88ec695ed4313b21192b1db914656acdc13e2741f
```

### 清单附件

[1451942.zip](https://github.com/user-attachments/files/16914293/1451942.zip)

### 备注

3288922513@qq.com

**Comment by pjy612** [comment]
* #258
